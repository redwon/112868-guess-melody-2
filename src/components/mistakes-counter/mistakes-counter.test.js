import React from 'react';
import renderer from 'react-test-renderer';

import MistakesCounter from './mistakes-counter';

describe(`MistakesCounter`, () => {
  it(`should not render mistakes`, () => {
    const tree = renderer.create(<MistakesCounter number={0} />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`should render mistakes`, () => {
    const tree = renderer.create(<MistakesCounter number={3} />).toJSON();

    expect(tree).toMatchSnapshot();
  });

});
