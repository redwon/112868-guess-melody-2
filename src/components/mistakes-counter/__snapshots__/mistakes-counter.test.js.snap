// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MistakesCounter should not render mistakes 1`] = `
<div
  className="game__mistakes"
/>
`;

exports[`MistakesCounter should render mistakes 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
</div>
`;
